bootstrap/components/sinks/kafka-sink-topology-component.json

```diff
@@ -35,13 +35,23 @@
         "hint": "schema,override"
       },
       {
-        "uiName": "Writer schema version",
+        "uiName": "Writer Schema Branch",
+        "fieldName": "schemaBranch",
+        "isOptional": true,
+        "tooltip": "Schema branch to be used fetching for writer schema versions. Default branch is MASTER",
+        "type": "enumstring",
+        "options": [],
+        "hint": "schema,override",
```

Comment:
We probably need to change the hint to schemaBranch and add a dependOn-topic hint here so UI can fetch all the branches for a given topic/schema.